cloudformation/tag-manager.yaml

```diff
@@ -284,6 +282,69 @@ Resources:
         Value: flexible
       - Key: App
         Value: tag-manager
+  TagManagerCloudfrontCachePolicy:
+    Type: "AWS::CloudFront::CachePolicy"
+    Properties:
+      CachePolicyConfig:
+        DefaultTTL: 86400
```

Comment:
Do you really want the default TTL to be 86400?

Reply:
I borrowed those numbers from the presence snapshot and assumed they derived from gu/cdk for a sensible reason I didn't know - shall I make it 3600?

Reply:
Now set at 3600.